docs/contributing.md

```diff
@@ -20,9 +20,10 @@ If you want to contribute and don't know where to start, here is a step-by-step
 
 1. Install [Node.js](https://nodejs.dev/learn/how-to-install-nodejs)
 2. Install [Yarn](https://yarnpkg.com/) `npm i -g yarn`
```

Comment:
Could you please change here
```
Enable [Corepack](https://nodejs.org/api/corepack.html) `corepack enable`
```

Reply:
ohh that doesn't exist in node version we are using, that means its currently useless

Reply:
and it has experimental status

Reply:
For volta users it needs also `volta install corepack` ;-)